### PR TITLE
Add Start and Stop tracking events.

### PR DESCRIPTION
**Pull request by gegy1000**

### CODE REVIEW

File: src/main/java/xyz/openmodloader/event/impl/PlayerEvent.java [hunk 1]
```diff
@@ -134,4 +135,62 @@ public static float handle(EntityPlayer player, ItemStack result, float xp) {
...
+         */
+        public Entity getTracking() {
+            return tracking;
+        }
+    }
+
+    /**
+     * This event is fired when a player starts tracking an entity.
+     */
+    public static class Start extends Track {
```
Comment (iLexiconn) [inline src/main/java/xyz/openmodloader/event/impl/PlayerEvent.java]: Put inside `Track` please

File: src/main/java/xyz/openmodloader/event/impl/PlayerEvent.java [hunk 2]
```diff
@@ -134,4 +135,62 @@ public static float handle(EntityPlayer player, ItemStack result, float xp) {
             return Math.min(1.0F, event.xp);
         }
     }
+
+    /**
+     * The base class for entity-tracking related events.
+     */
+    public static class Track extends PlayerEvent {
```
Comment (iLexiconn) [inline src/main/java/xyz/openmodloader/event/impl/PlayerEvent.java]: Rename to `Tracking`

**Comment by iLexiconn** [comment]
Does it work on multiplayer?

**Comment by gegy1000** [comment]
Yes.

File: src/main/java/xyz/openmodloader/event/impl/PlayerEvent.java [hunk 3]
```diff
@@ -134,4 +135,62 @@ public static float handle(EntityPlayer player, ItemStack result, float xp) {
...
+
+        /**
+         * The entity being tracked by the player.
+         */
+        protected final Entity tracking;
+
+        /**
+         *
```
Comment (iLexiconn) [inline src/main/java/xyz/openmodloader/event/impl/PlayerEvent.java]: add desc or remove whiteline

**Pull request merged**